src/routes/Landing.tsx

```diff
@@ -0,0 +1,338 @@
+import React from "react";
+import { RouteComponentProps } from "@reach/router";
+import { Flex, Box, Text, Logo, Button } from "components";
+import styled, { createGlobalStyle, keyframes } from "styled-components";
+import { themeGet } from "@styled-system/theme-get";
+import underline from "assets/underline.svg";
+import retrotoolScreenshot from "assets/retrotool-screenshot.png";
+import { Heart } from "styled-icons/fa-solid/Heart";
+import { Mobile } from "styled-icons/icomoon/Mobile";
+import { LocationOff } from "styled-icons/material/LocationOff";
+import { Cached } from "styled-icons/material/Cached";
+import { LogInCircle } from "styled-icons/boxicons-regular/LogInCircle";
+
+const HeartIcon = styled(Heart)`
+  width: 10px;
+  height: 10px;
+  margin: 0 6px;
+`;
+
+interface Props extends RouteComponentProps {}
+
+const LandingGlobalStyle = createGlobalStyle`
+  body {
+    background: ${themeGet("colors.white")} !important;
+  }
+`;
+
+const Section = styled(Box).attrs({ as: "section" })``;
+
+const underlineAnimation = keyframes`
+  from { width: 0; }
+  to { width: 100%; }
+`;
+
+const Underline = styled(Box)`
+  position: absolute;
+  bottom: -8px;
+  left: 0;
+  width: 0;
+  height: 8px;
+  background: url(${underline});
+  background-repeat: repeat-x;
+  background-position: center bottom;
+  background-size: 200px 8px;
+  animation: ${underlineAnimation} 0.3s 3s ease-in-out forwards;
+`;
+
+const Screenshot = styled(Box)`
+  position: absolute;
+  left: 80ch;
+  bottom: -20px;
+  width: 1275px
+  height: 670px;
+  background: url(${retrotoolScreenshot}) no-repeat;
+  filter: drop-shadow(0 0 1px rgba(0,0,0,.5));
+`;
+
+const RetroSteps = styled(Text).attrs({
+  as: "h3",
+  textStyle: "landingHowtoStep"
+})`
+  ::before {
+    content: "";
+    position: absolute;
+    margin-top: 12px;
+    margin-left: -40px;
+    width: 30px;
+    height: 1px;
+    background: ${themeGet("colors.violet")};
+  }
+`;
+
+const Landing: React.FC<Props> = () => (
+  <>
+    <LandingGlobalStyle />
+    <Box bg="white" margin="auto">
+      <Flex
+        as="header"
+        justifyContent="space-between"
+        alignItems="center"
+        p={6}
+      >
+        <Logo />
+        <Flex>
+          <Text mr={3}>
+            <a href="https://twitter.com/marciobarrios" tabIndex={3}>
```

Comment:
lol, I guess it's good for a start 😄

Reply:
Oh fuck! Jajaja should we create a twitter account?

Reply:
let's see what @retro-tool/all think, I think an email address should be enough